Week02/lesson03/homework/relaxr_starter_code/css/style.css

```diff
@@ -0,0 +1,225 @@
+/*
+Project Name: Blank Template
+Client: Your Client
+Author: Your Name
+Developer @GA in NYC
+*/
+
+
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+     -moz-box-sizing: border-box; /* Firexfox */
+     -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+     box-sizing: border-box; /* IE */
+}
+
+/* Clear fix hack */
+.clearfix:after {
+     content: ".";
+     display: block;
+     clear: both;
+     visibility: hidden;
+     line-height: 0;
+     height: 0;
+}
+
+.clear {
+  clear: both;
+}
+
+.alignright {
+  float: right;
+  padding: 0 0 10px 10px; /* note the padding around a right floated image */
+}
+
+.alignleft {
+  float: left;
+  padding: 0 10px 10px 0; /* note the padding around a left floated image */
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
+body {
+     color: #000;
+     font-size: 12px;
+     line-height: 1.4;
+     font-family: 'Open Sans', sans-serif;
+     rgb(134,141,157);
+
+}
+
+
+/******************************************
+/* LAYOUT
+/*******************************************/
+
+/* Center the container */
+#container {
+  width: 100%;
+  margin: auto;
+
+
+}
+
+/*I stuggled to make the layout work with the bk img in the body so had to move it into the header!*/
+
+header {
+    background: url(../images/header_bg.jpg) no-repeat center top;
+    background-size: cover;
+}
+
+.headerinner{
+  width: 75%;
+  margin-left: auto;
+  margin-right: auto;
+  padding-top: 20px;
+  color: #fff;
+  height: 700px;
+}
+
+
+.cta{
+  padding-top: 200px;
+  text-align: center;
+  font-weight: 600;
+  font-size: 24px;
+}
+
+.benefits{
+  background-color: #f0efef;
+  padding-top: 50px;
+  padding-bottom: 50px;
+}
+
+.benefitsinner{
+  width: 75%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.testimonial{
+  background: url(../images/testimonial_bg.jpg) no-repeat center top;
+  background-size: cover;
+  padding-top: 50px;
+  padding-bottom: 50px;
+}
+
+.testimonialinner{
+  width: 375px;
+  margin: auto;
+  color: #fff;
+  text-align: center;
+}
+
+.signup{
+  background-color: #f9e42e;
+  padding-top: 25px;
+  padding-bottom: 25px;
+  text-align: center;
+}
+
+
+footer {
+  background-color: #121212;
+  padding-top: 50px;
+  padding-bottom: 25px;
+  text-align: center;
+
+}
+
+
+
+/******************************************
+/* ADDITIONAL STYLES
+/*******************************************/
+
+.logo {
+  font-weight: 800;
+  font-size: 24px;
+}
+
+.button{
+  display: inline-block;
+  font-weight: 800;
+  padding: 10px 40px;
+  text-decoration: none;
+  border-radius: 5px 5px 5px 5px;
+  -moz-border-radius: 5px 5px 5px 5px;
+  -webkit-border-radius: 5px 5px 5px 5px;
```

Comment:
Good job using `border-radius` for the button, one small comment: because CSS reads top-to-bottom, you always want the non-prefixed version of a CSS property to be last, so __should__ the browser supports that property __without__ the need of a prefixed version (which if you recall browser makers use to try out experimental features), then the standardised, non-prefixed version will overwrite the old prefixed one, like so:

```css
-moz-border-radius: 5px 5px 5px 5px;
-webkit-border-radius: 5px 5px 5px 5px;
border-radius: 5px 5px 5px 5px; /* This one, the standardised one will overwrite the experimental ones above it */
```